code/modules/vore/fluffstuff/custom_boxes_vr.dm

```diff
@@ -217,6 +217,19 @@
 		/obj/item/weapon/melee/fluffstuff/wolfgirlsword,
 		/obj/item/weapon/shield/fluff/wolfgirlshield)
 
+// Victor Zisthus: KA-BOOM
+/obj/item/weapon/storage/box/fluff/kaboom
+	name = "KA-BOOM's Stuff"
+	desc = "A box filled with some starting equipment for the android."
+	has_items = list(/obj/item/weapon/spacecash/c500,
+	/obj/item/weapon/spacecash/c200 = 2,
```

Comment:
I think staff has come to a ruling that the thalers are a bit too much of an advantage, and NIFSofts that give mechanical benefits are not something that should be taken for granted.

Reply:
900 is also an oddly specific amount. This would be better handled by just tweaking wealth level in character setup, and if you're already maxed out then just ask folks if you can borrow the money or see if you can find other ways to earn it.